README.md

```diff
@@ -7,87 +8,127 @@
 [![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
 [![Twitter Follow](https://img.shields.io/twitter/follow/imagekitio?label=Follow&style=social)](https://twitter.com/ImagekitIo)
 
-ImageKit Python SDK allows you to use [image resizing](https://docs.imagekit.io/features/image-transformations), [optimization](https://docs.imagekit.io/features/image-optimization), [file uploading](https://docs.imagekit.io/api-reference/upload-file-api) and other [ImageKit APIs](https://docs.imagekit.io/api-reference/api-introduction) from applications written in the Python language.
+ImageKit Python SDK allows you to use [image resizing](https://docs.imagekit.io/features/image-transformations)
+, [optimization](https://docs.imagekit.io/features/image-optimization)
+, [file uploading](https://docs.imagekit.io/api-reference/upload-file-api) and
+other [ImageKit APIs](https://docs.imagekit.io/api-reference/api-introduction) from applications written in the Python
+language.
 
 Supported Python Versions: >=3.6
 
 Table of contents -
- * [Installation](#Installation)
- * [Initialization](#Initialization)
- * [URL Generation](#URL-generation)
- * [File Upload](#File-Upload)
- * [File Management](#File-Management)
- * [Utility Functions](#Utility-functions)
- * [Support](#Support)
- * [Links](#Links)
-
-
- ## Installation
- Go to your terminal and type the following command
+
+* [Installation](#Installation)
+* [Initialization](#Initialization)
+* [Versioning](#versioning)
+* [URL Generation](#URL-generation)
+* [File Upload](#File-Upload)
+* [File Management](#File-Management)
+* [Utility Functions](#Utility-functions)
+* [Handling errors](#handling-errors)
+* [Support](#Support)
+* [Links](#Links)
+
+## Installation
+
+Go to your terminal and type the following command
+
 ```bash
 pip install imagekitio
 ```
 
 ## Initialization
+
 ```python
 from imagekitio import ImageKit
+
 imagekit = ImageKit(
-    private_key='your private_key',
-    public_key='your public_key',
-    url_endpoint = 'your url_endpoint'
+    private_key='your_private_key',
```

Comment:
The code snippet is not formatted properly. Space before and after `=`. Please check all examples.